### ISSUE TITLE
Use of Python 2 shebangs for Python 3 sources

### ISSUE THREAD

**Issue by kosarev** [boriel/zxbasic#281]
On my seemingly standard Ubuntu setup running the .py files in the root of the repository ends with the following. The problem seems to be that they actually call the Python 2 interpreter whereas the sources look to be in Python 3. Replacing `python` with `python3` seems to resolve the problem.

```
$ ./zxb.py
Traceback (most recent call last):
  File "./zxb.py", line 16, in <module>
    import api.optimize
  File "/home/kosarev/labs/kosarev/github/zx/lab/repos/boriel/zxbasic/api/optimize.py", line 4, in <module>
    from ast_ import NodeVisitor
  File "/home/kosarev/labs/kosarev/github/zx/lab/repos/boriel/zxbasic/ast_/__init__.py", line 4, in <module>
    from .ast import Ast
  File "/home/kosarev/labs/kosarev/github/zx/lab/repos/boriel/zxbasic/ast_/ast.py", line 13, in <module>
    from .tree import Tree
  File "/home/kosarev/labs/kosarev/github/zx/lab/repos/boriel/zxbasic/ast_/tree.py", line 105
    yield from i.inorder()
             ^
SyntaxError: invalid syntax
```

**Comment by boriel** [comment]
Yes, I was precisely yesterday discussing this.
I think it's time to move to python3 shebangs since python2 is dead but it seems it will last in some linux boxes for a long time.

**Comment by kosarev** [comment]
Addressed in #282.
